tests/ci/env_helper.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+CI = bool(os.getenv("CI"))
```

Comment:
bad name, it's unclear what does it mean. Maybe better to have something like `IS_LOCAL_ENV` or `IS_CI_ENV`?

Reply:
That's a standard name, unfortunately